### PR TITLE
GitHub instructions changes

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
Some minor additions, style changes, and an additional link to the instructions for enabling/restricting CircleCI on GitHub.

# Reasons
We got feedback the instructions were hard to follow.

### CODE REVIEW

**Comment by devinmarieb** [comment]
The first commit is a lot because I changed the styling of the paragraph breaks.

File: jekyll/_cci2/gh-bb-integration.md [hunk 1]
```diff
@@ -10,67 +10,36 @@ version:
...
-CircleCI won't build. Refer to the
-[GitHub Edit a Hook document](https://developer.github.com/v3/repos/hooks/#edit-a-hook)
-or the
-[Atlassian Manage Webhooks document](https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html)
-for details.
-
-Refer to CircleCI documentation of [Workflows
-filters]({{ site.baseurl }}/2.0/workflows/#using-contexts-and-filtering-in-your-workflows)
-for how to build tag pushes.
+To use CircleCI you need to be using either GitHub or Bitbucket for your VCS. When you add a project to CircleCI, the following GitHub or Bitbucket Cloud settings are added to the repository using the permissions you gave CircleCI when you signed up:
```
Comment (thomasr8) [inline jekyll/_cci2/gh-bb-integration.md:22]:
```suggestion
To use CircleCI you need to use either GitHub or Bitbucket for your VCS. When you add a project to CircleCI, the following GitHub or Bitbucket Cloud settings are added to the repository using the permissions you gave CircleCI when you signed up:
```

File: jekyll/_cci2/gh-bb-integration.md [hunk 2]
```diff
@@ -10,67 +10,36 @@ version:
...
+
+- CircleCI processes PR hooks (Pull Request Hooks) to store PR information for the CircleCI app. If the Only Build Pull Requests setting is enabled, CircleCI will only trigger builds when a PR is opened, or when there is a push to a branch for which there is an existing PR. Even if this setting is enabled, CircleCI will always build all pushes to the project's default branch.
+- If the Build Forked Pull Requests setting is set, CircleCI will trigger builds in response to PRs created from forked repos.
+
+It is possible to edit the webhooks in GitHub or Bitbucket Cloud to restrict events that trigger a build. Editing the webhook settings lets you change which hooks get sent to CircleCI, but doesn't change the types of hooks that trigger builds. CircleCI will always build push hooks and will build on PR hooks (depending on settings), but if you remove push hooks from the webhook settings CircleCI won't build. Refer to the [GitHub Edit a Hook document](https://developer.github.com/v3/repos/hooks/#edit-a-hook) or the [Atlassian Manage Webhooks document](https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html) for details.
```
Comment (thomasr8) [inline jekyll/_cci2/gh-bb-integration.md:34]:
```suggestion
It is possible to edit the webhooks in GitHub or Bitbucket Cloud to restrict events that trigger a build. Editing the webhook settings lets you change which hooks get sent to CircleCI, but does not change the types of hooks that trigger builds. CircleCI will always build push hooks and will build on PR hooks (depending on settings), but if you remove push hooks from the webhook settings, CircleCI will not build. Refer to the [GitHub Edit a Hook document](https://developer.github.com/v3/repos/hooks/#edit-a-hook) or the [Atlassian Manage Webhooks document](https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html) for details.
```

File: jekyll/_cci2/gh-bb-integration.md [hunk 3]
```diff
@@ -10,67 +10,36 @@ version:
...
+
+Refer to CircleCI documentation of [Workflows filters]({{site.baseurl}}/2.0/workflows/#using-contexts-and-filtering-in-your-workflows) for how to build tag pushes.
```
Comment (thomasr8) [inline jekyll/_cci2/gh-bb-integration.md:36]:
```suggestion
Refer to the CircleCI documentation on [Workflows filters]({{site.baseurl}}/2.0/workflows/#using-contexts-and-filtering-in-your-workflows) for information on how to build tag pushes.
```

**Review by thomasr8 — APPROVED** [review]
@devinmarieb Looks great! Just a few minor suggestions.
